### PR TITLE
Fix failing build issues with the pyramid-fork.

### PR DESCRIPTION
**Pull request by jaycee**

### CODE REVIEW

**Comment by jaycee** [comment]
QA:

Ensure `make clean-all run` starts the server and that the gui is correctly served.

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -803,12 +803,12 @@
     },
     "less": {
```
Comment (hatched) [inline npm-shrinkwrap.json]: We still need to load less?

Comment (jaycee) [inline npm-shrinkwrap.json]: We do not; thought I had deleted all those references.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2714/
Test PASSed.

**Comment by hatched** [comment]
:+1: NO QA

I'm pretty sure all the less references were removed but I don't think that shrinkwrap will remove those deps.

**Comment by jaycee** [comment]
Thanks @hatched. Hoo-ray, shrinkwrap.

File: npm-shrinkwrap.json (hunk 1 above)
Comment (makyo) [inline npm-shrinkwrap.json]: I regenerated shrinkwrap as part of the test love, less was leftover in there, sorry for the confusion.

**Comment by bac** [comment]
Code :+1:

**Comment by bac** [comment]
QA-OK.

**Comment by jaycee** [comment]
Thanks @bac. Sorry for your git woes.

**Comment by jaycee** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jaycee** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/969

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**